src/get-docs.js

```diff
@@ -1,15 +1,26 @@
+const dir = require('node-dir')
 
 const getDocs = root => {
-    const path = require('path')
-    const fs = require('fs')
+    const mds = []
 
     return new Promise((resolve, reject) => {
-        fs.readFile(path.join(root, 'README.md'), 'utf8', (err, data) => {
-            if(err && err.code == 'ENOENT') return reject(new Error('NO_DOCS_AVAILABLE'))
+        dir.readFiles(root, {
+            match: /.md$/,
+            excludeDir: ['node_modules'],
+            exclude: /^\./
+        },
+        function(err, content, next) {
+            if (err) throw err
```

Comment:
Can we swap this for reject(err)? Using a throw statement here might break promise chaining/catching.

Reply:
Also, can we cover this bit with a unit test case, i.e. mocking readFiles to throw an exception here?